docs/admin/system-information.rst

```diff
@@ -1923,10 +1923,14 @@ tables. Any table-specific health issues, exposed by the
 +----------------------------+-------------------------------------+--------------+
 | ``description``            | A description of the current health | ``TEXT``     |
 +----------------------------+-------------------------------------+--------------+
-| ``missing_shards``         | The number of unassigned or not     | ``INTEGER``  |
+| ``pending_tasks``          | The number of pending tasks on the  | ``BIGINT``   |
+|                            | master node for updating the        |              |
```

Comment:
maybe: `master node, which can potentially update the cluster state`
Not sure if it's better, to me: `for updating` reads a bit weird.

Reply:
what about:
`The number of pending cluster state update tasks`?

Reply:
yep, nice, thx!